src/patternfly/components/Card/card.scss

```diff
@@ -7,16 +7,21 @@
 
   // Card Header variables
   --pf-c-card__header--PaddingTop: var(--pf-global--spacer--lg);
-  --pf-c-card__header--PaddingX: var(--pf-global--spacer--lg);
+  --pf-c-card__header--PaddingRight: var(--pf-global--spacer--lg);
   --pf-c-card__header--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-card__header--PaddingLeft: var(--pf-global--spacer--lg);
 
   // Card Body variables
-  --pf-c-card__body--PaddingY: var(--pf-global--spacer--sm);
-  --pf-c-card__body--PaddingX: var(--pf-global--spacer--lg);
+  --pf-c-card__body--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-card__body--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-card__body--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-card__body--PaddingTop: var(--pf-global--spacer--sm);
```

Comment:
Probably move PaddingTop above PaddingRight so it goes Top, Right, Bottom, Left like elsewhere.

Reply:
nice find, updated